src/improvements/tasks/MultisigTask.sol

```diff
@@ -1050,9 +1050,64 @@ abstract contract MultisigTask is Test, Script {
 
 abstract contract L2TaskBase is MultisigTask {
     using EnumerableSet for EnumerableSet.AddressSet;
+    using stdToml for string;
+
+    struct StateOverride {
+        address target;
+        KeyValueOverride keyValueOverride;
+    }
+
+    struct KeyValueOverride {
+        bytes32 slot;
+        bytes32 value;
+    }
```

Comment:
nit, but it might make sense to just call this `StorageOverride` like in [base-contracts](https://github.com/ethereum-optimism/base-org-contracts/blob/7cca28be0f94134f9de1c6f409779b57d5f269b7/script/universal/Simulation.sol#L19-L22), to minimize differences.

Reply:
good call: https://github.com/ethereum-optimism/superchain-ops/pull/727/commits/007e2702f8b187925c3b54aed091c391f1c57985